src/layer/text.js

```diff
@@ -101,6 +101,15 @@ var Text = function(parentEl) {
         return true;
     };
 
+    this.highlightIndentGuides = true;
```

Comment:
other places use $ to mark variables like this as private `this.$highlightIndentGuides = true;`